spec/1_base.adoc

```diff
@@ -182,6 +182,14 @@ See <<gpkg_spatial_ref_sys_sql>>.
 
 [[gpkg_srs_table_data_values]]
 ====== Table Data Values
+The `srs_id` column is a primary key for this table and this primary key is used as a foreign key throughout GeoPackage.
```

Comment:
The value of `srs_id` in geometries must be a `srs_id` value (Req 33). This is not a foreign key in the relational sense. It could be worthwhile to mention it in this clarification. E.g.

 > The `srd_id` column values are also found in the `srs_id` attribute of the GeoPackage SQL Geometry Binary Format.